adapters/yieldone/yieldone.go

```diff
@@ -0,0 +1,125 @@
+package yieldone
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/mxmCherry/openrtb"
+	"github.com/prebid/prebid-server/adapters"
+	"github.com/prebid/prebid-server/errortypes"
+	"github.com/prebid/prebid-server/openrtb_ext"
+)
+
+type YieldoneAdapter struct {
+	endpoint string
+}
+
+// MakeRequests makes the HTTP requests which should be made to fetch bids.
+func (a *YieldoneAdapter) MakeRequests(request *openrtb.BidRequest, reqInfo *adapters.ExtraRequestInfo) ([]*adapters.RequestData, []error) {
+	var errors = make([]error, 0)
+
+	for i := 0; i < len(request.Imp); i++ {
+		preprocess(&request.Imp[i])
```

Comment:
The `placementId` param declared in `static/bidder-params/yieldone.json` is not being used by this adapter because it doesn't get unmarshaled neither inside the `MakeRequests` nor the `preprocess` methods